packages/history-service/src/base-history.ts

```diff
@@ -5,70 +5,54 @@ export interface ConsumerHistory extends history.History {
   destroy(): void;
 }
 
-function setRootHistoryKey(
-  consumerLocation: history.Location,
-  rootHistoryKey: string | undefined
-): void {
-  consumerLocation.state = {...consumerLocation.state, rootHistoryKey};
+export interface ConsumerHistoryStates {
+  readonly [consumerId: string]: unknown;
 }
 
 export abstract class BaseHistory implements ConsumerHistory {
```

Comment:
at this point in time we do not need this base class (yagni ;)

Reply:
as discussed, I will add the static history in this PR